pkg/cable/vxlan/vxlan.go

```diff
@@ -140,7 +140,7 @@ func (v *vxlan) createVxlanInterface(activeEndPoint string, port int) error {
 
 	v.vxlanIface.vtepIP = vtepIP
 
-	err = v.addIPRule()
+	err = v.configureIPRule(Add)
```

Comment:
Please include the following in the **Commit** message. You added it in the PR, but it should also be in the Commit message. Also, the convention we are using is "Fixes" 

Fixes: https://github.com/submariner-io/submariner-operator/issues/1913

Reply:
Thanks for the details, I'll update the commit message.
I copy/paste the 'Resolves' from an operator PR [1], is there a different syntax for operator and submariner repos?

[1]
https://github.com/submariner-io/submariner-operator/pull/1929

Reply:
The syntax is the same across all repositories. We haven’t documented a preference, as far as I’m aware, but as Sridhar says we seem to mostly use “Fixes”. See [the GitHub documentation](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) for details.

Some statistics from the operator repository: “Resolves:” appears 8 times, “Fixes:” 65 times, and various other keywords appear many other times (“Resolves-Issue:”, “Resolves issue”, “Fixes-Issue:”, “Fixes issue”, “Closes” etc.).

Reply:
I see, I will use 'Fixes'  from now on.
Thanks Stephen